cli/pub_command.go

```diff
@@ -115,6 +117,9 @@ Available template functions are:
 	req.Flag("raw", "Show just the output received").Short('r').Default("false").BoolVar(&c.raw)
 	req.Flag("header", "Adds headers to the message").Short('H').StringsVar(&c.hdrs)
 	req.Flag("count", "Publish multiple messages").Default("1").IntVar(&c.cnt)
+	req.Flag("replies", "Wait for multiple replies from services. 0 waits until timeout").Default("1").IntVar(&c.replyCount)
+	req.Flag("reply-timeout", "Maximum timeout between incoming replies.").Default("250ms").DurationVar(&c.replyTimeout)
```

Comment:
If we are making this default we need to somehow explain to users that we are using this and not --timeout in the UX of the application.

As I mentioned earlier I think for interacting with nats server APIs this would be a good default - they respond immediately - but we have no idea what services users interact with and this default could be confusing them not knowing why the default timeout is being ignored for example. Even the example in the docs again wttr using nats reply would not be reliable since that API is far latency wise and might not respond immediately.  

So not sure, we should maybe warn at the end when early terminating based on prefering that value over --timeout

Reply:
Per our slack convo, I'll update the subsequent timeout to do 300ms + response time for the first.  So, for example if the first response came in at 100ms, then we'd wait 400ms (100ms + 300ms) for the 2nd, resulting a total of 500ms for the entire operation.

Reply:
Update, will use a rtt average.